### PR TITLE
Arcana mod series 2

### PR DESCRIPTION
**Pull request by chaosvolt**
Second part of the arcana mod PRing. This adds everything except the curious structure, spirit of fire, demon claw, and upright weapon.

Third Essence:
- Dull essence is obtained by dismantling magic items and essence using the correct tool. Used to craft and fuel certain items.

Item Dismantling:
- Recipes for dismantling most magical items, using the hammer of the hunter. Items made by The Cleansing Flame lack deconstruct recipes, as dismantling unsanctioned magical items is implied to be their shtick.

Magic Items:
- Hammer of the hunter. Weapon of choice for magiphobes the world over. Required for uncrafting most magical items and essence. Also can be used to blind enemies, and yourself if you aren't careful.
- Incorruptible Sword. A holy blade that uses "consecrated" dull essence, when activated it gives of light and searing radiance.
- Gilded Aegis. A cloak that can be activated, consuming dull essence to heal a smattering of ailments, in exchange for pain and fatigue.
- Wraithslayer crossbow. Fires dull essence, basically a magic crossbow with good armor penetration and that lovely "shot" flag to annoy those annoying hard-to-shoot monsters.

Summoning Items:
- Silver glyph. On its own it's merely a shiny piece of silver. But...
- Glyph of trickery. Ever wanted your own pet kreck?
- Glyph of impassiveness. Probably the only summoning item that won't get your face wrecked if you mess up.
- Glyph of the beholder. It's a thing of beauty, isn't it?
- Glyph of Crawling Chaos. Sit. Stay. Good boy! Who's a good widdle winged worm wons...er, monster?
- Glyph of anthozoa. No, not the name of some alien deity.
- Glyph of Yuggoth. Don't ask them about their two-for-one plastic surgery and genetic alteration special.
- Glyph of the platyhelminthes. In Soviet Russia, spawn of Ythogtha is bribed by you!
- Glyph of the Elder Things. Tekeli-li? Tekeli-li.

Books:
- The Cleansing Flame. Levels 4-7. Written by an order of witch-hunters, it allows crafting a select number of sanctioned items fueled by dull essence.
- To Master the Unknown. Levels 6-10. A book for summoning your very own pet monstrosities.

Misc:
- Addition of dull essence to the chalice recipe for making normal essence, at reduced efficiency compared to sacrificing blood essence.

### CODE REVIEW

**Comment by chaosvolt** [comment]
Hmm. Had an oddball idea. A "powered welding goggles" instance of tool-armor, specifically to synergize with the blinding effect of the hammer of the hunter. Thoughts on that?

**Comment by chaosvolt** [comment]
Ah, thank you for the merge.

**Comment by DavidKeaton** [comment]
Love the mods mate, keep em up!

File: data/mods/Arcana/tools.json [hunk 1]
```diff
@@ -155,8 +151,405 @@
...
+        "water_extinguish_message": "The sword shimmers in the water, before the glow fades.",
+        "noise": 0,
+        "noise_chance": 100
+        }
+    ],
+    "category" : "weapons",
+    "qualities": [["CUT", 1], ["BUTCHER", 9]]
+  },{
+        "type":"TOOL",
+        "id" : "hexenhammer",
```
Comment (DavidKeaton) [inline data/mods/Arcana/tools.json]: id and hexen, how apropos!

**Comment by chaosvolt** [comment]
Reference overdose much? Though the hammer isn't itself a Hexen reference. A hammer that shoots exploding hammers would be amusing though. XP

**Comment by DavidKeaton** [comment]
Oh I know it wasn't, just a big Doom fan :-P a hammer that shot hammers that exploded into exploding chainguns perhaps?

**Comment by chaosvolt** [comment]
Huehuehue. Amusing as that would be, now I'm tempted to see how to make the Hammer of Retribution in a logical way.

Just making it a hammer that counts as a gun that shoots explosive rounds would be the easiest way, but something that functions more like veinreaver (only with some way to not constantly have to pick it up) could also work.

**Pull request merged**